pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1400,6 +1400,8 @@ type GatewaySpec struct {
 type ZoneSpec struct {
 	// RGW Zone the Object Store is in
 	Name string `json:"name"`
+	// Endpoint For Zone and Zonegroup Endpoints List
+	Endpoint string `json:"endpoint"`
```

Comment:
```suggestion
        // +optional
	Endpoint string `json:"endpoint,omitempty"`
```

It's optional right?